CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 NativeScript CLI Changelog
 ================
 
+1.1.0 (2015, June 10)
+==
+
+### Breaking
+
+* Replaced `.tnsproject` with `package.json` to let you use npm and work with custom plugins in your project.<br/>The first time you run any NativeScript command on an existing project, your `.tnsproject` file will be automatically transformed into a `package.json` file describing your project configuration and data. After this, make sure to commit and push your changes if you are using version control.<br/>After the transformation is complete, you can install and use custom NativeScript plugins inside your project.
+
+### New
+
+* [Implemented #510](https://github.com/NativeScript/nativescript-cli/issues/510): You can work with custom NativeScript plugins in your project. You can add a plugin from an npm package, a local folder, a URL or a `.tar.gz` file. For more information, run `$ tns help plugin`
+* [Implemented #362](https://github.com/NativeScript/nativescript-cli/issues/362): You can use npm as package manager for your NativeScript projects.
+	
+### Updated
+
+* [Updated #358](https://github.com/NativeScript/nativescript-cli/issues/358): Multiple improvements and bug fixes for iOS debugging.
+
+### Fixed
+
+* [Fixed #446](https://github.com/NativeScript/nativescript-cli/issues/446): Autocompletion might continue to work after disabling becauce `tns autocomplete disable` does not update `.profile` files.
+* [Fixed #391](https://github.com/NativeScript/nativescript-cli/issues/391): The default 9-patch image for Android projects is malformed and causes error messages in the build log.
+* [Fixed #324](https://github.com/NativeScript/nativescript-cli/issues/324): When you run commands for the `ios` platform, the NativeScript CLI might showUnable to find adb in PATH. Default one from %s resources will be used. the following unrelated adb warning: `Unable to find adb in PATH. Default one from %s resources will be used.`
```

Comment:
"showUnable to find adb in PATH. Default one from %s resources will be used. the" looks like a typo